### ISSUE TITLE
IdsInput: Accessibility WCAG Guideline: 1.3.3 Sensory Characteristics: Information is conveyed in the form of symbol.

### ISSUE THREAD

**Issue by CindyMercadoReyes** [infor-design/enterprise-wc#1408]
Describe the bug : Information is conveyed in the form of symbol.

To Reproduce
"1. Launch the URL: https://main.wc.design.infor.com/ids-input/example.html
2. Navigate to '*' symbol and verify"
3. Information is conveyed in the form of symbol. '*' symbol is used to declare a field as required. But it is not written anywhere that '*' symbol means required field.

Expected behavior : Information should not be conveyed in the form of symbol. If it is conveyed then it should be written in textual form for eg. * means required field.

User Impact:  User will not be able to identify the purpose of that symbol.

Screenshots  If applicable, add screenshots to help explain your problem.: https://app.screencast.com/eDktytJ8iTEAP

Dev Recommendation:  Information about that '*' symbol should be present in textual form. For eg. * means required field

Platform
NVDA/Windows

Additional context
NA

**Comment by tmcconechy** [comment]
I dont see `''symbol and verify"` anywhere on this page. But i know about the problem of using `*` but that has become common place. So think we can try to fix this by reading out `required` or `required *` and add to the docs if needed.
